File: Com.Fubei.OpenApi.Vb/Biz/AgentApi/Model/Request/ARefundOrderParam.vb
Imports Com.Illuminati.Galileo.Model
Imports Newtonsoft.Json

Namespace Biz.V2.Model.Request
    Public Class ARefundOrderParam
        Inherits BaseEntity
        
        <JsonProperty("merchant_id")>
        Public Property MerchantId As Integer?

        <JsonProperty("order_sn")>
        Public Property OrderSn As String

        <JsonProperty("merchant_order_sn")>
        Public Property MerchantOrderSn As String

        <JsonProperty("ins_order_sn")>
        Public Property InsOrderSn As String

        <JsonProperty("merchant_refund_sn")>
        Public Property MerchantRefundSn As String

        <JsonProperty("refund_amount")>
        Public Property RefundAmount As Decimal

        <JsonProperty("handler")>
        Public Property Handler As Integer?
    End Class

End Namespace

File: Com.Fubei.Api.OpenApi/Biz/AgentApi/Model/Request/ARefundOrderParam.cs
using Com.Illuminati.Galileo.Model;
using Newtonsoft.Json;

namespace Com.Fubei.Api.OpenApi.Biz.AgentApi.Model.Request
{
    public class ARefundOrderParam : BaseEntity
    {
        [JsonProperty("merchant_id")]
        public int? MerchantId { get; set; }

        [JsonProperty("order_sn")]
        public string OrderSn { get; set; }

        [JsonProperty("merchant_order_sn")]
        public string MerchantOrderSn { get; set; }

        [JsonProperty("ins_order_sn")]
        public string InsOrderSn { get; set; }

        [JsonProperty("merchant_refund_sn")]
        public string MerchantRefundSn { get; set; }

        [JsonProperty("refund_amount")]
        public decimal RefundAmount { get; set; }

        [JsonProperty("handler")]
        public int? Handler { get; set; }
    }
}
